src/material/tooltip/testing/tooltip-harness.ts

```diff
@@ -52,6 +52,11 @@ export abstract class _MatTooltipHarnessBase extends ComponentHarness {
     return !!panel && !(await panel.hasClass(this._hiddenClass));
   }
 
+  /** Gets whether the tooltip is disabled */
+  async isDisabled(): Promise<boolean> {
+    return !(await this._optionalPanel());
```

Comment:
This doesn't tell you that the tooltip is disabled, it tells you whether it's opened.

Reply:
I took a look at `tooltip.ts` and that's the only way I could think of to find out about the `disabled` state.
https://github.com/angular/components/blob/main/src/material/tooltip/tooltip.ts#L420

you are right in a sense that even if the tooltip isn't **disabled** but has an empty message, this method will return `true`. 
But I think it's kinda correct to return `true` in that case too. cause at the end, a tooltip with an empty message doesn't have a difference with a disabled one.

Can you think of any other way to handle it?

Reply:
Usually the way we'd handle it is to add something to the DOM (e.g. a class) that reflects the disabled state.

Reply:
Done